src/Microsoft.AspNetCore.Routing/project.json

```diff
@@ -20,6 +20,7 @@
     "xmlDoc": true
   },
   "dependencies": {
+    "Microsoft.AspNetCore.Hosting": "1.2.0-*",
```

Comment:
@davidfowl There is an already an existing extension on IApplicationBuilder which can be used. I updated the PR.